cartpost-service/src/main/java/com/example/cartpostservice/common/exception/CustomStatusCode.java

```diff
@@ -14,6 +14,7 @@ public enum CustomStatusCode {
 
     FORBIDDEN_ITEM(HttpStatus.FORBIDDEN, 6403, "소유하고 있는 아이템이 아닙니다"),
     FORBIDDEN_COMMISSION(HttpStatus.FORBIDDEN, 6413, "의뢰글에 접근 권한이 없습니다"),
+    NOT_OPEN_COMMISSION(HttpStatus.UNAUTHORIZED, 6423, "중단된 의뢰글은 다시 모집 공고를 열 수 없습니다"),
```

Comment:
의뢰글을 열 수 없는 상황에 사용하는 상태 코드네요!
다만, 코드 이름만 보면 의뢰글이 아직 열리지 않은 상태라는 뜻으로 오해할 수 있을 것 같아요.
상태 확인이 아니라 동작 실패임을 직관적으로 나타내기 위해 아래와 같은 표현은 어떨지 제안드립니다!

```suggestion
    CANNOT_OPEN_COMMISSION(HttpStatus.UNAUTHORIZED, 6423, "중단된 의뢰글은 다시 모집 공고를 열 수 없습니다"),
```

Reply:
넵 조금 더 자연스러울 것 같네요